client/app/main/student/student.controller.js

```diff
@@ -2,18 +2,8 @@
 
 var app = angular.module('app');
 
-function StudentCtrl($scope, $stateParams, Intervention, Modal, Outreach,
-  Student, toastr) {
-
+function StudentCtrl($scope, $state, $stateParams, Student, toastr) {
   $scope.student = Student.get({id: $stateParams.id});
```

Comment:
Looks like you would need to add it to the StudentInterventionCtrl since it is already injected in the StudentOutreachCtrl.